samples/nrf9160/nrf_cloud_mqtt_multi_service/src/application.c

```diff
@@ -341,6 +345,9 @@ void main_application_thread_fn(void)
 		LOG_INF("Current date and time determined");
 	}
 
+	nrf_cloud_log_init();
+	nrf_cloud_log_send(LOG_LEVEL_INF, "Hello world.");
```

Comment:
can this be changed to something more interesting?
maybe some build info,  application version, mfwver, etc?